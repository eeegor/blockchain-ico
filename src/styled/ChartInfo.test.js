import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { ChartInfo } from '.';

describe('ChartInfo', () => {
	it('renders chart wrapper', () => {
		const tree = renderer.create(<ChartInfo>Info</ChartInfo>).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
