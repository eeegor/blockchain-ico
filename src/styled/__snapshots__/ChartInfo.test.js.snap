// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartInfo renders chart wrapper 1`] = `
.c0 {
  position: absolute;
  top: 12px;
  left: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 12px 0;
  width: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c0 .c1 {
  margin-bottom: 0;
  color: #fff;
  font-size: 75%;
  opacity: 0.5;
}

@media (min-width:767px) {
  .c0 {
    padding: 24px 0 12px 0;
  }
}

<div
  className="c0"
>
  Info
</div>
`;
